app/events/applications/application_deleted_event.rb

```diff
@@ -9,7 +9,8 @@ def self.create(application)
       metadata: {
         provider_id: application.provider_account_id || application.tenant_id,
         zync: {
-          service_id: application.service_id
+          service_id: application.service_id,
+          proxy_id: application.service.proxy.id # TODO: It could not exist anymore!
```

Comment:
If we want to have this PR quickly, then for now this should be `application.service&.proxy&.id` and in another PR fixing the issue that at this point the proxy could not exist anymore :smile: